### PR TITLE
language modifications for jenkins kube master/slave cleanup

### PR DESCRIPTION
**Pull request by gaurav-nelson**
for PR #3175

### CODE REVIEW

File: using_images/other_images/jenkins.adoc [hunk 1]
```diff
@@ -386,7 +388,8 @@ Also note the `contrib` and `contrib/bin` subdirectories.  They allow for the in
 scripts for your image.
 
 The Jenkins image also provides auto-discovery and auto-configuration
-of slave images for the Kubernetes plug-ins by scanning the project Jenkins is deployed in for
+of slave images for the 
+Kubernetes plug-ins by scanning the project. Jenkins is deployed in for
```
Comment (gaurav-nelson) [inline using_images/other_images/jenkins.adoc]: @gabemontero Original sentence(below) is not clear. Can you please check?
> The Jenkins image also provides auto-discovery and auto-configuration
>  +of slave images for the Kubernetes plug-ins by scanning the project Jenkins is deployed in for
>   existing image streams with the label *role* set to *jenkins-slave*.		  existing image streams with the label *role* set to *jenkins-slave*.

Comment (gabemontero) [inline using_images/other_images/jenkins.adoc]: @gaurav-nelson your proposed change does not sound right to me.  How about this:

```
... of slave images for the Kubernetes plug-in.  The search for such images is done by examining existing image streams within the project that Jenkins is running in.  Specifically, the search looks for images streams that have the label *role* set to *jenkins-slave*.
```

**Comment by gaurav-nelson** [comment]
@bfallonf :beer: p(b)eer review please!

File: using_images/other_images/jenkins.adoc [hunk 2]
```diff
@@ -362,22 +362,24 @@ https://wiki.jenkins-ci.org/display/JENKINS/Kubernetes+Plugin[Kubernetes
 plug-in] that allows Jenkins slaves to be dynamically provisioned on multiple
 container hosts using Kubernetes and {product-title}.
 
-To facilitate the using of the Kubernetes plug-in, {product-title} provides three images
-suitable for use as Jenkins slaves..
+To use the Kubernetes plug-in, {product-title} provides three images
+suitable for use as Jenkins slaves; the *_Base_*, *_Maven_*, and *_NodeJS_* images.
```
Comment (bfallonf) [inline using_images/other_images/jenkins.adoc]: @gaurav-nelson I'd suggest the semi-colon above turn into a colon.

File: using_images/other_images/jenkins.adoc [hunk 3]
```diff
@@ -362,22 +362,24 @@ https://wiki.jenkins-ci.org/display/JENKINS/Kubernetes+Plugin[Kubernetes
...
 
 The first is a https://github.com/openshift/jenkins/tree/master/slave-base[base image] for Jenkins slaves.
```
Comment (bfallonf) [inline using_images/other_images/jenkins.adoc]: As the nest little section is a list, perhaps end the sentence above with a colon as well.

File: using_images/other_images/jenkins.adoc [hunk 4]
```diff
@@ -362,22 +362,24 @@ https://wiki.jenkins-ci.org/display/JENKINS/Kubernetes+Plugin[Kubernetes
...
 
-Two images which extend this base image are also provided:
+ - It pulls in both the required tools (headless Java, the Jenkins JNLP client) and the useful ones
+(including git, tar, zip, nss among others). 
+ - It establishes the JNLP slave agent as the entrypoint.
+ - It includes the oc client tooling for invoking command line operations from within Jenkins jobs, and
+ - It provides Dockerfiles for both Centos and RHEL images.
 
-* https://github.com/openshift/jenkins/tree/master/slave-maven[Maven]
+Two additional images, that extends the base image are also provided, namely
```
Comment (bfallonf) [inline using_images/other_images/jenkins.adoc]: s/,namely/:

File: using_images/other_images/jenkins.adoc [hunk 5]
```diff
@@ -362,22 +362,24 @@ https://wiki.jenkins-ci.org/display/JENKINS/Kubernetes+Plugin[Kubernetes
...
+
+* https://github.com/openshift/jenkins/tree/master/slave-maven[Maven], and
 * https://github.com/openshift/jenkins/tree/master/slave-nodejs[NodeJS]
 
 Both the Maven and NodeJS slave images are configured as Kubernetes Pod Tempate images within the {product-title} Jenkins image's
-configuration for the Kubernetes plugin.  That configuration includes labels for each which can
+configuration for the Kubernetes plugin.  That configuration includes labels for each of the images that can
```
Comment (bfallonf) [inline using_images/other_images/jenkins.adoc]: Double space inbetween the sentences above.

File: using_images/other_images/jenkins.adoc [hunk 6]
```diff
@@ -386,14 +388,17 @@ Also note the `contrib` and `contrib/bin` subdirectories.  They allow for the in
 scripts for your image.
 
 The Jenkins image also provides auto-discovery and auto-configuration
-of slave images for the Kubernetes plug-ins by scanning the project Jenkins is deployed in for
-existing image streams with the label *role* set to *jenkins-slave*.
+of slave images for the 
+Kubernetes plug-in. The search for such images is done by examining existing image streams within the project that Jenkins is running in. The search specifically looks for image streams that have the label `role` set to `jenkins-slave`.
```
Comment (bfallonf) [inline using_images/other_images/jenkins.adoc]: "The search for such images is done by examining existing image streams within the project that Jenkins is running in."

is passive. Perhaps:

"The Jenkins image also provides auto-discovery and auto-configuration of the slave images for the Kubernetes plug-ins. The Jenkins image searches for these in the existing image streams within the project that it is running in."

**Comment by bfallonf** [comment]
@gaurav-nelson A few comments. LGTM otherwise!

**Pull request merged**